src/pages/PatientPersonalInfo.js

```diff
@@ -16,9 +20,19 @@ const PatientPersonalInfo = ({
   setPage,
   page,
 }) => {
+  const heightInFeet =
+    (intakeState.height &&
+      intakeState.height.split(`'`)[0].replace(/[^0-9]/g, "")) ||
+    0;
+  const heightInInch =
+    (intakeState.height &&
+      intakeState.height.split(`'`)[1].replace(/[^0-9]/g, "")) ||
+    0;
```

Comment:
would be better to move this to an `useEffect` so that they don't run on every re-render